pom.xml

```diff
@@ -16,7 +16,10 @@
   <properties>
     <jenkins.version>1.609</jenkins.version>
     <java.level>6</java.level>
-    <workflow.version>1.4.3</workflow.version>
+    <workflow-job.version>1.4.3</workflow-job.version>
+    <workflow-cps.version>1.4.3</workflow-cps.version>
```

Comment:
No, minimum `workflow-cps` 2.10 is required.
I have added them as separate properties to be able to test it like this:
https://github.com/jenkinsci/ws-cleanup-plugin/pull/30/files#diff-99947b5d4e31828f2c311a58853445a6R340

`mvn clean install -Djenkins.version=1.642.1 -Djava.level=7 -Dworkflow-job.version=2.4 -Dworkflow-basic-steps.version=2.1 -Dworkflow-cps.version=2.10 -Dworkflow-durable-task-step.version=2.4`